CRM/Upgrade/Incremental/php/FourSix.php

```diff
@@ -220,4 +223,19 @@ public static function fixCaseLog(CRM_Queue_TaskContext $ctx) {
     return TRUE;
   }
 
+  public function upgrade_4_6_3($rev) {
```

Comment:
@totten Are the docs out of date? http://wiki.civicrm.org/confluence/display/CRMDOC/Upgrade+Reference#UpgradeReference-IncrementalSQLFiles
Is this function no longer required?